paymentsheet/build.gradle

```diff
@@ -92,4 +92,4 @@ dependencies {
     androidTestImplementation "androidx.test:runner:$androidTestVersion"
     androidTestUtil "androidx.test:orchestrator:$androidTestVersion"
 
-}
```

Comment:
probably enable [ktlint](https://github.com/stripe/stripe-android/blob/c9604e15a63bfe771f07ed1a2cea356c6aa7fb04/payments-core/build.gradle#L134) in a follow up so these changes are captured